examples/conditional-send/conditional_send.md

```diff
@@ -0,0 +1,7 @@
+# Conditional send
+
+The send action in workers can be used within a conditional context, allowing for more flexible and dynamic inter-worker communication based on specific conditions.
```

Comment:
```suggestion
The send action in workers can be used in a conditional context, allowing for more flexible and dynamic inter-worker communication based on specific conditions.
```

Include something about the receive side too, since it becomes aware that there may not be a message.